### ISSUE TITLE
delta_C

### ISSUE THREAD

**Issue by yarimitsu** [quantifish/isotope#8]
I adjusted the Delta_C parameter to something more widely supported in the literature (i.e., mean of 0.4/TL increase, SD 1.3). This parameter is tricky, because results from laboratory studies are fairly variable (hence the large SD in review papers). 

Would it make sense to use a uniform prior here instead? Or perhaps allow tau (trophic level, derived from 15N data) to inform this parameter by taking into account individual variability (i.e., reinstate C_tot = delta_C * tau)? This would effectively change u_C to u_C_i, i = 1, ..., N.

**Comment by quantifish** [comment]
That's fine Yumi. I suggest against playing about with priors for now. As it stands your model relies almost entirely on the priors as the posterior is not very different in most cases.

I am finding it very hard to follow what is going in still though, I don't know if the figures are for the new model or the old model, I suggest deleting all figures related to the old model. We really need a decent suite of plots showing the model posteriors, as both trace plots and as histograms with the priors plotted on top. The obs v exp plot you have done is good.
